File: SoftUni - Advanced/Defining Classes/Defining Classes - Exercise/09. Pokemon Trainer/Pokemon.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _9.Pokemon_Trainer
{
    public class Pokemon
    {
        public string  Name { get; set; }

        public string Element { get; set; }

        public int Health { get; set; }

        public Pokemon(string name, string element, int health)
        {
            Name = Name;
            Element = element;
            Health = health;
        }
    }
}

File: SoftUni - Java Advanced/Defining Classes/Define Classes - Exercise/_06_Pokémon_Trainer/Pokemon.java
package exercise._06_Pokémon_Trainer;

public class Pokemon {
    private String name;
    private String element;
    private int health;

    public Pokemon(String name, String element, int health) {
        this.name = name;
        this.element = element;
        this.health = health;
    }

    public void decreaseHealthPokemon() {
        this.health -= 10;
    }

    public int getHealth() {
        return health;
    }

    public void setHealth(int health) {
        this.health = health;
    }

    public String getElement() {
        return element;
    }
}
